index.html

```diff
@@ -215,217 +588,409 @@ <h3>Code</h3>
+        <div class="grid grid-cols-2 gap-8 mt-8">
+          <div
+            class="flex flex-col items-center justify-center mx-auto transition-colors bg-green-100 rounded-lg cursor-pointer hover:bg-green-200 size-40 grid-stack-item-content grid-stack-item newWidget"
+          >
+            <ion-icon
+              name="add-circle"
+              class="text-4xl fill-green-700"
+            ></ion-icon>
+            <p class="text-center text-green-700">
+              Drag me into the dashboard!
+            </p>
           </div>
-          <div class="text-center card text-white grid-stack-item newWidget">
-            <div class="card-body grid-stack-item-content">
-              <div>
-                <ion-icon name="add-circle" style="font-size: 400%"></ion-icon>
-              </div>
-              <div>
-                <span>Drag me into the dashboard!</span>
-              </div>
-            </div>
+          <div
+            id="trash"
+            class="flex flex-col items-center justify-center mx-auto bg-red-100 rounded-lg size-40"
+          >
+            <ion-icon name="trash" class="text-4xl fill-red-700"></ion-icon>
+
+            <p class="text-center text-red-700">Drop here to remove widget!</p>
           </div>
         </div>
-        <div class="col-sm-12 col-md-10" style="padding-bottom: 25px;">
-          <div class="grid-container">
-            <div class="grid-stack" id="advanced-grid"></div>
-          </div>
+
+        <div
+          class="p-1 mt-8 border rounded-lg bg-gradient-to-br from-gray-200 to-gray-50"
+        >
+          <div class="grid-stack" id="advanced-grid"></div>
         </div>
-      </div>
-    </section>
-    <section id="usedBy" class="bg-info text-white">
-      <div class="row" style="margin-bottom: 15px;">
-        <div class="col-lg-12">
-          <ion-icon name="desktop" class="float-right" style="font-size: 350%"></ion-icon>
-
-          <h1>Why use it</h1>
-          <hr />
-          <ul>
-            <li><a href="https://www.npmjs.com/package/gridstack" target="_blank">npm downloads</a></li>
-            <li><a href="https://www.formget.com/jquery-layout-plugins/" target="_blank">6 Best jQuery Layout Plugins 2020</a></li>
-          </ul>
-          <h1>Used by</h1>
-          <hr />
+      </section>
+
+      <section id="usedBy" class="pt-40 mx-auto max-w-7xl">
+        <p
+          class="inline px-4 py-1 font-medium tracking-tight border border-gray-700 rounded-full"
+        >
+          Articles
+        </p>
+        <h2 class="mt-4 text-4xl font-medium">Why use gridstack.js ?</h2>
+        <p class="mt-4 font-light">
+          Find out why developers everywhere are choosing gridstack.js for their
+          projects.
+        </p>
+        <ul class="mt-8">
+          <li>
+            <a
+              href="https://www.npmjs.com/package/gridstack"
+              target="_blank"
+              class="hover:text-indigo-500"
+              >npm downloads</a
```

Comment:
those links don't look clickable in general (no blue hint). hard to tell what's a section title, vs links.